disttask/importinto/dispatcher.go

```diff
@@ -205,29 +210,29 @@ func (h *flowHandle) ProcessNormalFlow(ctx context.Context, handle dispatcher.Ta
 			}
 			metaBytes = append(metaBytes, bs)
 		}
-		gTask.Step = Import
+		gTask.Step = StepImport
 		return metaBytes, nil
-	case Import:
-		defer func() {
-			if err == nil {
-				err = finishJob(ctx, handle, taskMeta, &importer.JobSummary{ImportedRows: taskMeta.Result.LoadedRowCnt})
-			} else {
-				// todo: we're not running in a transaction with task update, there might be case
-				// failJob return error, but task update succeed.
-				if err2 := failJob(ctx, handle, taskMeta, err.Error()); err2 != nil {
```

Comment:
Why do we remove this logic?

Reply:
i thought framework will call ProcessErrFlow and handle it there previously, will change it

Reply:
https://github.com/pingcap/tidb/pull/44616/commits/8b4e8f3ba4f2e7a679f02347e09f68178395ea51

Reply:
Got it.